unit_tests/falco/app/actions/test_configure_interesting_sets.cpp

```diff
@@ -301,9 +301,10 @@ TEST(ConfigureInterestingSets, selection_custom_base_set)
 		// note: `accept` is not included even though it is matched by the rules,
 		// which means that the custom negation base set has precedence over the
 		// final selection set as a whole
-		"connect", "open", "ptrace", "mmap", "execve", "read", "syncfs", "sched_process_exit"
+		// todo(jasondellaluce): add "accept4" once names_to_sc_set is polished on the libs side
```

Comment:
When disabling `accept`, Falco also disables `accept4`. That's still expected because I still have to clean up `names_to_sc_set` on libsinsp.